docs/docs/installation.md

```diff
@@ -75,3 +69,12 @@ task build
 ```
 
 After this the binary is available on path `./bin/jalapeno`.
+
+## Updating your jalapeno version
+
+### Homebrew
+
+```bash tab={"label":"Homebrew"}
```

Comment:
This doesn't do anything _yet_, because there is only one "tab code block" mentioned.